### PR TITLE
Clarify what is meant by "personal data".

### PR DESCRIPTION
**Pull request by msporny**
This PR is an attempt to partially address issue #93 by clarifying what is meant by "personal data".

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/controller-document/pull/105.html" title="Last updated on Oct 19, 2024, 8:56 PM UTC (84dd981)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/controller-document/105/de03d6e...84dd981.html" title="Last updated on Oct 19, 2024, 8:56 PM UTC (84dd981)">Diff</a>

### CODE REVIEW

**Review by iherman — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -3009,10 +3009,15 @@ <h2>Keep Personal Data Private</h2>
 
       <p>
 If a [=controller document=] is about a specific individual and is
-public-facing, it is <em>critical</em> that the [=controller documents=]
-contain no personal data. Personal data can instead be transmitted through other
-means such as 1) verifiable credentials [[?VC-DATA-MODEL-2.0]], or 2)
-other private communication channels.
+public-facing, it is <em>critical</em> that the [=controller document=] contains
+no personal biometric or biographical data. While it is true that personal data
+might include pseudonymous information, such as a public cryptographic key,
```
Comment (David-Chadwick) [inline index.html:3014]:
```suggestion
might include pseudonymous information, such as a public cryptographic key or an IP address, 
```

**Review by David-Chadwick — APPROVED** [review]
Subject to minor editorial

**Review by dlongley — APPROVED** [review]

File: index.html [hunk 2]
```diff
@@ -3009,10 +3009,15 @@ <h2>Keep Personal Data Private</h2>
 
       <p>
 If a [=controller document=] is about a specific individual and is
-public-facing, it is <em>critical</em> that the [=controller documents=]
-contain no personal data. Personal data can instead be transmitted through other
-means such as 1) verifiable credentials [[?VC-DATA-MODEL-2.0]], or 2)
-other private communication channels.
+public-facing, it is <em>critical</em> that the [=controller document=] contains
```
Comment (TallTed) [inline index.html:3012]:
```suggestion
public-facing, it is <em>critical</em> that [=controller documents=] contain
```

File: index.html [hunk 3]
```diff
@@ -3009,10 +3009,15 @@ <h2>Keep Personal Data Private</h2>
...
+no personal biometric or biographical data. While it is true that personal data
+might include pseudonymous information, such as a public cryptographic key,
+publishing that sort of information does not create the same immediate privacy
+dangers as publishing an individual's full name, profile photo, or social media
+account in a [=controller document=]. As an alternative, personal data can
+instead be transmitted through other means such as verifiable credentials
+[[?VC-DATA-MODEL-2.0]] or other data formats sent over private and secure
+communication channels.
```
Comment (TallTed) [inline index.html:3020]:
```suggestion
account in a [=controller document=]. A better alternative is to transmit
such personal data through other means such as verifiable credentials
[[?VC-DATA-MODEL-2.0]] or other data formats sent over private and secure
communication channels.
```

**Review by TallTed — CHANGES_REQUESTED** [review]
Small stuff.

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**